scripts/automation/commandlint/run.py

```diff
@@ -19,8 +19,644 @@
 from azure.cli.core.help_files import helps
 
 
+WHITE_LIST_COMMANDS = [
```

Comment:
Move the white list to a separate configuration file. Like `allowed-error.txt`. It is important to separate the data from the algorithm.